operator_ui/src/pages/Bridges/Index.test.js

```diff
@@ -1,76 +1,88 @@
 /* eslint-env jest */
-import createStore from 'createStore'
-import { ConnectedIndex as Index } from 'pages/Bridges/Index'
-import { mount } from 'enzyme'
-import bridgesFactory from 'factories/bridges'
+
 import React from 'react'
-import { Provider } from 'react-redux'
-import { MemoryRouter } from 'react-router-dom'
-import clickNextPage from 'test-helpers/clickNextPage'
-import clickPreviousPage from 'test-helpers/clickPreviousPage'
-import syncFetch from 'test-helpers/syncFetch'
+import {
+  renderWithRouter,
+  screen,
+  waitFor,
+  waitForElementToBeRemoved,
+} from 'test-utils'
+import userEvent from '@testing-library/user-event'
 import globPath from 'test-helpers/globPath'
+import bridgesFactory from 'factories/bridges'
 
-const classes = {}
-const mountIndex = (opts = {}) =>
-  mount(
-    <Provider store={createStore()}>
-      <MemoryRouter>
-        <Index classes={classes} pageSize={opts.pageSize} />
-      </MemoryRouter>
-    </Provider>,
-  )
+import { ConnectedIndex as Index } from 'pages/Bridges/Index'
 
 describe('pages/Bridges/Index', () => {
-  it('renders the list of bridges', async () => {
-    expect.assertions(2)
+  const { getAllByRole, getByRole } = screen
 
+  it('renders the list of bridges', async () => {
     const bridgesResponse = bridgesFactory([
       {
         name: 'reggaeIsntThatGood',
         url: 'butbobistho.com',
       },
     ])
+
     global.fetch.getOnce(globPath('/v2/bridge_types'), bridgesResponse)
 
-    const wrapper = mountIndex()
+    renderWithRouter(<Index pageSize={1} />)
 
-    await syncFetch(wrapper)
-    expect(wrapper.text()).toContain('reggaeIsntThatGood')
-    expect(wrapper.text()).toContain('butbobistho.com')
+    await waitFor(() => getByRole('heading'))
```

Comment:
```suggestion
    await findByRole('heading')
```
is equivalent and I think should be preferred